components/resources/ai_chat_prompts.grdp

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit-part>
+  <message name="IDS_AI_CHAT_SUMMARIZE_PROMPT" translateable="false" desc="Default prompt for summarization">
+    Write a concise summary and always respond in bullet points:
```

Comment:
Well, maybe we could have our proxy API accept the article text as an argument, but I'm not sure what it would gain us. After article summarization, we do allow the user to type in whatever question they want. So as long as we're allowing that, then all we're doing is forcing them to submit some content of a page along with their free-form prompt.

We also don't know if this is the final version of this feature - some people suggest we allow more free-form chatting without forcing the article summary part, so I didn't want to get us too tied to how we're doing it right now. Or maybe we'll go the other way and only allow some "suggested" questions to be asked. But we can definitely consider it either now if we're worried or when we collect feedback and decide how the prototype moves forward to product feature, or when we turn our proxy off because of too much traffic.

I do agree if all we're doing is summarization (and we had a good prompt that avoids commands breaking out from the content), then we should have our own API in front of the actual AI API.